docs/docusaurus/versioned_docs/version-1.7.0/dp/architecture_overview.md

```diff
@@ -111,15 +112,14 @@ The related request is also marked as processed.
 #### *Protocol Controller
 
 **NOTE**: This is a historical name for a component that is no longer an independent part of Domain Proxy, but still
-needs to be mentioned.
+needs to be mentioned. It was removed by that [GitHub PR](https://github.com/magma/magma/pull/12420)
```

Comment:
```suggestion
needs to be mentioned. It was removed by this [GitHub PR](https://github.com/magma/magma/pull/12420).
```

Reply:
(One should avoid `this/that` in such phrases whenever possible.)[https://github.com/magma/magma/wiki/Contributing-Code-Conventions#all-documentation] How about `It was removed by [PR #12420](...link...).`.
The full stop should still be added though.

Reply:
Rephrased and removed `this/that`.

Reply:
Looks good